example/package.json

```diff
@@ -65,8 +65,8 @@
     "@types/react-dom": "^16.9.4",
     "@types/reactstrap": "^8.0.6",
     "@types/yup": "^0.29.7",
-    "@typescript-eslint/eslint-plugin": "^4.2.0",
-    "@typescript-eslint/parser": "^4.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
```

Comment:
This error shows up when trying to start the example app: 

> Error in Plugin "@typescript-eslint" was conflicted between "../.eslintrc.yaml » eslint-config-availity/workflow#overrides[0]" and "BaseConfig » eslint-config-availity/workflow#overrides[0]".

Haven't looked in to why yet but thinking they may have changed some default rules?

Reply:
good catch: 6d28343f22e057b0da4cc7db1077bec1c8a9d8ae

The rules for the workflow and the example project conflicted, so I set the the `root` to `true` for the example project so it doesn't look in the parent directory for configuration options. https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy